test/topology_custom/test_view_build_status.py

```diff
@@ -216,3 +216,68 @@ async def test_view_build_status_migration_to_v2(request, manager: ManagerClient
 
     result = await cql.run_async("SELECT * FROM system.view_build_status_v2")
     assert len(result) == 6
+
+# Start with view_build_status v1 mode, and create entries such that
+# some of them correspond to removed nodes or non-existent views.
+# Then migrate to v2 table and verify that only valid entries belonging to known nodes
+# and views are migrated to the new table.
+@pytest.mark.asyncio
+async def test_view_build_status_migration_to_v2_with_cleanup(request, manager: ManagerClient):
+    # First, force the first node to start in legacy mode
+    cfg = {'force_gossip_topology_changes': True}
+
+    servers = [await manager.server_add(config=cfg)]
+    # Enable raft-based node operations for subsequent nodes - they should fall back to
+    # using gossiper-based node operations
+    del cfg['force_gossip_topology_changes']
+
+    # We start with total 4 nodes and we will remove one of them before the migration.
+    servers += [await manager.server_add(config=cfg) for _ in range(3)]
```

Comment:
Nit: `servers += await manager.servers_add(3, config=cfg)`

Reply:
it doesn't work. I think I can't add servers concurrently because I'm starting in gossip mode

Reply:
Right, sorry, forgot that the test starts in gossip mode.